phpcs.xml.dist

```diff
@@ -8,7 +8,7 @@
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra" />
 	<rule ref="WordPress.WP.I18n"/>
-	<config name="text_domain" value="performance-lab,default,auto-sizes"/>
+	<config name="text_domain" value="performance-lab,default,auto-sizes,speculation-rules"/>
```

Comment:
This will need to be adjusted once #1002 is merged, instead a `plugins/speculation-rules/phpcs.xml.dist` file should be added then.